nixos/modules/hardware/tuxedo-drivers.nix

```diff
@@ -3,13 +3,20 @@
 with lib;
 
 let
-  cfg = config.hardware.tuxedo-keyboard;
-  tuxedo-keyboard = config.boot.kernelPackages.tuxedo-keyboard;
+  cfg = config.hardware.tuxedo-drivers;
+  tuxedo-drivers = config.boot.kernelPackages.tuxedo-drivers;
 in
   {
-    options.hardware.tuxedo-keyboard = {
+    imports = [
+      (mkRenamedOptionModule [ "hardware" "tuxedo-keyboard" "enable" ] [ "hardware" "tuxedo-drivers" "enable" ])
+    ];
+
+    options.hardware.tuxedo-drivers = {
       enable = mkEnableOption (lib.mdDoc ''
```

Comment:
I think lib.mdDoc should no longer be used, see https://nixos.org/manual/nixpkgs/unstable#function-library-lib.options.mdDoc

Reply:
(you can just remove it completely I think)

Reply:
done